framework/Cluster.js

```diff
@@ -12,8 +12,32 @@
  * @param {object} socket
  * @class
  */
-function Cluster(id, vehicles, commodities, socket) {
+function PFCluster(id, vehicles, commodities, socket) {
+    var routeSubscriptionMsg = {
+        "routeSubscribe": {
+            "model": "Cluster",
+            "id": id
+        }
+    };
 
+    setTimeout(function(){
+        socket.send(JSON.stringify(routeSubscriptionMsg));
+    }, 1250);
```

Comment:
This is probably the the worst line of code in the entire project right now. should use socket.onOpen instead of waiting a set amount of time